velox/expression/fuzzer/ExpressionFuzzer.cpp

```diff
@@ -253,6 +253,17 @@ static const std::unordered_map<
             /// them to fuzzer instead of hard-coding signatures here.
             getSignaturesForCast(),
         },
+        {
+            "concat_ws",
```

Comment:
nit: maybe include a note indicating that this signature is only for Spark SQL. We will separate the Presto and Spark signatures in https://github.com/facebookincubator/velox/pull/9949. Thanks.

Reply:
@rui-mo, just updated. Thanks!